### PR TITLE
Release: decK 1.4.0

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
Bump version for decK 1.4.0.

Added new flags: `konnect-token` and `konnect-token-file`. Will update Konnect guide properly in a follow-up update, to add instructions on how to use PATs with decK.

### Reason
https://github.com/Kong/deck/releases/tag/v1.14.0

### Testing
Netlify

### CODE REVIEW

File: src/deck/guides/konnect.md [hunk 1]
```diff
@@ -32,6 +32,15 @@ This takes precedence over the `--konnect-password-file` flag.
 `--konnect-runtime-group-name`
 :  {{site.konnect_short_name}} runtime group name.
 
+{% if_version gte:1.14.x %}
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/guides/konnect.md:37]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/guides/konnect.md [hunk 2]
```diff
@@ -32,6 +32,15 @@ This takes precedence over the `--konnect-password-file` flag.
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/guides/konnect.md:41]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck.md [hunk 3]
```diff
@@ -69,6 +69,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck.md:75]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck.md [hunk 4]
```diff
@@ -69,6 +69,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck.md:79]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_completion.md [hunk 5]
```diff
@@ -151,6 +151,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_completion.md:157]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_completion.md [hunk 6]
```diff
@@ -151,6 +151,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_completion.md:161]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_convert.md [hunk 7]
```diff
@@ -92,6 +92,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_convert.md:98]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_convert.md [hunk 8]
```diff
@@ -92,6 +92,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_convert.md:102]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_diff.md [hunk 9]
```diff
@@ -125,6 +125,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_diff.md:131]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_diff.md [hunk 10]
```diff
@@ -125,6 +125,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_diff.md:135]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_dump.md [hunk 11]
```diff
@@ -116,6 +116,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_dump.md:122]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_dump.md [hunk 12]
```diff
@@ -116,6 +116,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_dump.md:126]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_ping.md [hunk 13]
```diff
@@ -82,6 +82,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_ping.md:88]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_ping.md [hunk 14]
```diff
@@ -82,6 +82,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_ping.md:92]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_reset.md [hunk 15]
```diff
@@ -108,6 +108,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_reset.md:114]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_reset.md [hunk 16]
```diff
@@ -108,6 +108,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_reset.md:118]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_sync.md [hunk 17]
```diff
@@ -119,6 +119,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_sync.md:125]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_sync.md [hunk 18]
```diff
@@ -119,6 +119,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_sync.md:129]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_validate.md [hunk 19]
```diff
@@ -114,6 +114,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_validate.md:120]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_validate.md [hunk 20]
```diff
@@ -114,6 +114,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_validate.md:124]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

File: src/deck/reference/deck_version.md [hunk 21]
```diff
@@ -78,6 +78,17 @@ You may also need to pass in as header the User-Agent that was used to create th
 
 {% endif_version %}
 
+{% if_version gte:1.14.x %}
+
+`--konnect-token`
+:  Personal Access Token associated with your {{site.konnect_short_name}} account, this takes precedence over `--konnect-token-file` flag.
```
Comment (Guaris) [inline src/deck/reference/deck_version.md:84]:
```suggestion
:  Personal access token associated with your {{site.konnect_short_name}} account, this takes precedence over the `--konnect-token-file` flag.
```

File: src/deck/reference/deck_version.md [hunk 22]
```diff
@@ -78,6 +78,17 @@ You may also need to pass in as header the User-Agent that was used to create th
...
+
+
+`--konnect-token-file`
+:  File containing the Personal Access Token to your {{site.konnect_short_name}} account.
```
Comment (Guaris) [inline src/deck/reference/deck_version.md:88]:
```suggestion
:  File containing the personal access token to your {{site.konnect_short_name}} account.
```

**Review by Guaris — CHANGES_REQUESTED** [review]

**Comment by Guaris** [comment]
Ignore those vale errors IMO

**Comment by lena-larionova** [comment]
> Ignore those vale errors IMO

Definitely planning to. We're going to need to adjust the config at some point to stop triggering on files that aren't being changed...

**Review by Guaris — APPROVED** [review]

**Pull request merged**